specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json

```diff
@@ -4469,6 +4469,302 @@
         }
       }
     },
+    "subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/virtualMachines/{instanceId}/extensions/{vmExtensionName}": {
```

Comment:
this one caused Semantic Error. The path should start with "/", such as "/subscriptions/{subscriptionId}/..."